### DISCUSSION TITLE
Unable to download data through wget

### DISCUSSION THREAD

**Discussion by clairevania** [MIT-LCP/mimic-code#1614] [MIMIC-IV-CXR]
I am unable to use the wget to download the mimic cxr data. It asks me for password which I enter and then I get this message. What should I do ?

--2023-07-31 15:51:27--  https://physionet.org/files/mimic-cxr/2.0.0/
Resolving physionet.org (physionet.org)... 18.18.42.54
Connecting to physionet.org (physionet.org)|18.18.42.54|:443... connected.
HTTP request sent, awaiting response... 401 Unauthorized
Authentication selected: Basic realm="PhysioNet", charset="UTF-8"
Reusing existing connection to physionet.org:443.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [text/html]
physionet.org/files/mimic-cxr/2.0.0: No such file or directory
physionet.org/files/mimic-cxr/2.0.0/index.html: No such file or directory

Cannot write to ‘physionet.org/files/mimic-cxr/2.0.0/index.html’ (Success).

**Comment by AdvancingEther** [comment]
I met the same question, have you solved yet?
